configure

```diff
@@ -2354,6 +2370,9 @@ fi
 if test "$disable_opt" = "yes" ; then
   output_sym "CONFIG_DISABLE_OPTIMIZATIONS"
 fi
+if test "$valgrind_valgrind_h" = "yes"; then
+  output_sym "HAVE_VALGRIND_VALGRIND_H"
```

Comment:
Perhaps this should be CONFIG_VALGRIND_DEV as mentioned by Jens. I'm also wondering if we should just roll the drd header check into it too as it seems likely you would have both and it saves another compile test...

Reply:
I will rename HAVE_VALGRIND_VALGRIND_H into CONFIG_VALGRIND_DEV and I will also combine the two header checks into a single check.